bedrock/firefox/templates/firefox/family/index.html

```diff
@@ -14,7 +14,9 @@
 {% block page_title_prefix %}The Tech Talk{% endblock %}
 {% block page_title_suffix %}{% endblock %}
 
-{% block page_image %}{{ static('protocol/img/logos/firefox/browser/og.png') }}{% endblock %}
+{% block page_image %}{{ static('img/firefox/family/meta-image.jpg') }}{% endblock %}
+{% block page_og_title %}Firefox for families{% endblock %}
```

Comment:
The `page_og_title` block normally draws from the complete page title, which comprise the blocks `page_title_prefix`, `page_title`, and `page_title_suffix`. If there's a reason we want the OpenGraph data to differ this is fine, but otherwise this should go in `page_title` and the og bits just match automatically.

Reply:
@maureenlholland - do you know if they wanted a different social share title than page title? I just didn't want to override the existing page title while I was working on this specific thing

Reply:
Yep, we had a discussion about this in the [Creative deck](https://docs.google.com/presentation/d/1jjd96ODHV_lbdQCFg7BWcFjN5TXu-KzcuFvdKutvCeo/edit#slide=id.g144d4c3ce2c_2_19)

From a creative perspective, since the meta image contains text, they were hoping to avoid being repetitive with the meta title.

From an SEO perspective, it's usually best if the title matches the H1 of the page, but that doesn't really matter in this case because "tech talk" and "firefox for families" aren't premium search terms.

From an a11y perspective ([according to Deque](https://dequeuniversity.com/checklists/web/page-title)), it's best practice if the title matches the H1.

Let's keep them separate for now. If needed, we can remove the og_title block, and "The Tech Talk" can serve as title to page and preview.